src/format/index.js

```diff
@@ -1,4 +1,4 @@
-import dateFnsFormat from 'date-fns/format/index.js'
+import { format as dateFnsFormat } from 'date-fns/format.js'
```

Comment:
```suggestion
import { format as dateFnsFormat } from 'date-fns'
```
To avoid any further path change, maybe we should refer to global package ? It works and it's not path-specific, so more secure (imho).

Reply:
Thought about this as well, but when I try to do this then I get errors.. 🤔 

```sh
$ yarn test

START:
ℹ ｢wdm｣: Built at: 01/08/2024 8:58:09 AM
ℹ ｢wdm｣: Compiled successfully.
ℹ ｢wdm｣: Compiling...
✖ ｢wdm｣: Built at: 01/08/2024 8:58:11 AM
Entrypoint ../test = ../test.js
[./node_modules/date-fns/index.mjs] 8.8 KiB {../test} [built]
[./node_modules/date-fns/locale/en-GB.js] 772 bytes {../test} [built]
[./node_modules/power-assert/index.js] 2.2 KiB {../test} [built]
[./src sync recursive \/test\.js$] ./src sync \/test\.js$ 460 bytes {../test} [built]
[./src/_lib/newDateUTC/test.js] 36.4 KiB {../test} [optional] [built]
[./src/_lib/tzIntlTimeZoneName/test.js] 22.4 KiB {../test} [optional] [built]
[./src/_lib/tzParseTimezone/test.js] 127 KiB {../test} [optional] [built]
[./src/_lib/tzTokenizeDate/test.js] 31.8 KiB {../test} [optional] [built]
[./src/format/test.js] 171 KiB {../test} [optional] [built]
[./src/formatInTimeZone/test.js] 16.4 KiB {../test} [optional] [built]
[./src/getTimezoneOffset/test.js] 100 KiB {../test} [optional] [built]
[./src/toDate/test.js] 255 KiB {../test} [optional] [built]
[./src/utcToZonedTime/test.js] 53.5 KiB {../test} [optional] [built]
[./src/zonedTimeToUtc/test.js] 89.8 KiB {../test} [optional] [built]
[./test.js] 124 bytes {../test} [built]
    + 510 hidden modules

WARNING in ./node_modules/power-assert-formatter/lib/create.js 30:28-49
Critical dependency: the request of a dependency is an expression
 @ ./node_modules/power-assert-formatter/index.js
 @ ./node_modules/power-assert/index.js
 @ ./src/getTimezoneOffset/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers.mjs 96:9-27
Can't import the named export 'AMPMMidnightParser' from non EcmaScript module (only default export is available)
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers.mjs 95:9-19
Can't import the named export 'AMPMParser' from non EcmaScript module (only default export is available)
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers.mjs 89:9-19
Can't import the named export 'DateParser' from non EcmaScript module (only default export is available)
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse.mjs 352:23-49
Can't import the named export 'DateToSystemTimezoneSetter' from non EcmaScript module (only default export is available)
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers.mjs 90:9-24
Can't import the named export 'DayOfYearParser' from non EcmaScript module (only default export is available)
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers.mjs 91:9-18
Can't import the named export 'DayParser' from non EcmaScript module (only default export is available)
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers.mjs 97:9-24
Can't import the named export 'DayPeriodParser' from non EcmaScript module (only default export is available)
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers.mjs 78:9-18
Can't import the named export 'EraParser' from non EcmaScript module (only default export is available)
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers.mjs 82:9-27
Can't import the named export 'ExtendedYearParser' from non EcmaScript module (only default export is available)
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers.mjs 104:9-31
Can't import the named export 'FractionOfSecondParser' from non EcmaScript module (only default export is available)
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers.mjs 100:9-24
Can't import the named export 'Hour0To11Parser' from non EcmaScript module (only default export is available)
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers.mjs 99:9-24
Can't import the named export 'Hour0to23Parser' from non EcmaScript module (only default export is available)
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers.mjs 101:9-24
Can't import the named export 'Hour1To24Parser' from non EcmaScript module (only default export is available)
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers.mjs 98:9-24
Can't import the named export 'Hour1to12Parser' from non EcmaScript module (only default export is available)
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers.mjs 94:9-21
Can't import the named export 'ISODayParser' from non EcmaScript module (only default export is available)
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers.mjs 106:9-26
Can't import the named export 'ISOTimezoneParser' from non EcmaScript module (only default export is available)
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers.mjs 105:9-31
Can't import the named export 'ISOTimezoneWithZParser' from non EcmaScript module (only default export is available)
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers.mjs 88:9-22
Can't import the named export 'ISOWeekParser' from non EcmaScript module (only default export is available)
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers.mjs 81:9-26
Can't import the named export 'ISOWeekYearParser' from non EcmaScript module (only default export is available)
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers.mjs 92:9-23
Can't import the named export 'LocalDayParser' from non EcmaScript module (only default export is available)
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers.mjs 87:9-24
Can't import the named export 'LocalWeekParser' from non EcmaScript module (only default export is available)
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers.mjs 80:9-28
Can't import the named export 'LocalWeekYearParser' from non EcmaScript module (only default export is available)
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers.mjs 102:9-21
Can't import the named export 'MinuteParser' from non EcmaScript module (only default export is available)
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers.mjs 85:9-20
Can't import the named export 'MonthParser' from non EcmaScript module (only default export is available)
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers.mjs 83:9-22
Can't import the named export 'QuarterParser' from non EcmaScript module (only default export is available)
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers.mjs 103:9-21
Can't import the named export 'SecondParser' from non EcmaScript module (only default export is available)
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers.mjs 93:9-33
Can't import the named export 'StandAloneLocalDayParser' from non EcmaScript module (only default export is available)
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers.mjs 86:9-30
Can't import the named export 'StandAloneMonthParser' from non EcmaScript module (only default export is available)
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers.mjs 84:9-32
Can't import the named export 'StandAloneQuarterParser' from non EcmaScript module (only default export is available)
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers.mjs 108:9-36
Can't import the named export 'TimestampMillisecondsParser' from non EcmaScript module (only default export is available)
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers.mjs 107:9-31
Can't import the named export 'TimestampSecondsParser' from non EcmaScript module (only default export is available)
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers.mjs 79:9-19
Can't import the named export 'YearParser' from non EcmaScript module (only default export is available)
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers/YearParser.mjs 13:11
Module parse failed: Unexpected token (13:11)
File was processed with these loaders:
 * ./node_modules/babel-loader/lib/index.js
You may need an additional loader to handle the result of these loaders.
| // | AD 12345 | 12345 | 45 | 12345 | 12345 | 12345 |
| export class YearParser extends Parser {
>   priority = 130;
|   incompatibleTokens = ["Y", "R", "u", "w", "I", "i", "e", "c", "t", "T"];
|   parse(dateString, token, match) {
 @ ./node_modules/date-fns/parse/_lib/parsers.mjs 2:0-54 79:9-19
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers/EraParser.mjs 3:11
Module parse failed: Unexpected token (3:11)
File was processed with these loaders:
 * ./node_modules/babel-loader/lib/index.js
You may need an additional loader to handle the result of these loaders.
| import { Parser } from "../Parser.mjs";
| export class EraParser extends Parser {
>   priority = 140;
|   parse(dateString, token, match) {
|     switch (token) {
 @ ./node_modules/date-fns/parse/_lib/parsers.mjs 1:0-52 78:9-18
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers/AMPMMidnightParser.mjs 4:11
Module parse failed: Unexpected token (4:11)
File was processed with these loaders:
 * ./node_modules/babel-loader/lib/index.js
You may need an additional loader to handle the result of these loaders.
| import { dayPeriodEnumToHours } from "../utils.mjs";
| export class AMPMMidnightParser extends Parser {
>   priority = 80;
|   parse(dateString, token, match) {
|     switch (token) {
 @ ./node_modules/date-fns/parse/_lib/parsers.mjs 19:0-70 96:9-27
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers/AMPMParser.mjs 4:11
Module parse failed: Unexpected token (4:11)
File was processed with these loaders:
 * ./node_modules/babel-loader/lib/index.js
You may need an additional loader to handle the result of these loaders.
| import { dayPeriodEnumToHours } from "../utils.mjs";
| export class AMPMParser extends Parser {
>   priority = 80;
|   parse(dateString, token, match) {
|     switch (token) {
 @ ./node_modules/date-fns/parse/_lib/parsers.mjs 18:0-54 95:9-19
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers/FractionOfSecondParser.mjs 4:11
Module parse failed: Unexpected token (4:11)
File was processed with these loaders:
 * ./node_modules/babel-loader/lib/index.js
You may need an additional loader to handle the result of these loaders.
| import { mapValue, parseNDigits } from "../utils.mjs";
| export class FractionOfSecondParser extends Parser {
>   priority = 30;
|   parse(dateString, token) {
|     const valueCallback = value => Math.floor(value * Math.pow(10, -token.length + 3));
 @ ./node_modules/date-fns/parse/_lib/parsers.mjs 27:0-78 104:9-31
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers/QuarterParser.mjs 4:11
Module parse failed: Unexpected token (4:11)
File was processed with these loaders:
 * ./node_modules/babel-loader/lib/index.js
You may need an additional loader to handle the result of these loaders.
| import { parseNDigits } from "../utils.mjs";
| export class QuarterParser extends Parser {
>   priority = 120;
|   parse(dateString, token, match) {
|     switch (token) {
 @ ./node_modules/date-fns/parse/_lib/parsers.mjs 6:0-60 83:9-22
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers/StandAloneQuarterParser.mjs 4:11
Module parse failed: Unexpected token (4:11)
File was processed with these loaders:
 * ./node_modules/babel-loader/lib/index.js
You may need an additional loader to handle the result of these loaders.
| import { parseNDigits } from "../utils.mjs";
| export class StandAloneQuarterParser extends Parser {
>   priority = 120;
|   parse(dateString, token, match) {
|     switch (token) {
 @ ./node_modules/date-fns/parse/_lib/parsers.mjs 7:0-80 84:9-32
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers/ExtendedYearParser.mjs 4:11
Module parse failed: Unexpected token (4:11)
File was processed with these loaders:
 * ./node_modules/babel-loader/lib/index.js
You may need an additional loader to handle the result of these loaders.
| import { parseNDigitsSigned } from "../utils.mjs";
| export class ExtendedYearParser extends Parser {
>   priority = 130;
|   parse(dateString, token) {
|     if (token === "u") {
 @ ./node_modules/date-fns/parse/_lib/parsers.mjs 5:0-70 82:9-27
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers/DayOfYearParser.mjs 5:11
Module parse failed: Unexpected token (5:11)
File was processed with these loaders:
 * ./node_modules/babel-loader/lib/index.js
You may need an additional loader to handle the result of these loaders.
| import { isLeapYearIndex, parseNDigits, parseNumericPattern } from "../utils.mjs";
| export class DayOfYearParser extends Parser {
>   priority = 90;
|   subpriority = 1;
|   parse(dateString, token, match) {
 @ ./node_modules/date-fns/parse/_lib/parsers.mjs 13:0-64 90:9-24
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers/StandAloneMonthParser.mjs 5:11
Module parse failed: Unexpected token (5:11)
File was processed with these loaders:
 * ./node_modules/babel-loader/lib/index.js
You may need an additional loader to handle the result of these loaders.
| import { mapValue, parseNDigits, parseNumericPattern } from "../utils.mjs";
| export class StandAloneMonthParser extends Parser {
>   priority = 110;
|   parse(dateString, token, match) {
|     const valueCallback = value => value - 1;
 @ ./node_modules/date-fns/parse/_lib/parsers.mjs 9:0-76 86:9-30
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers/TimestampMillisecondsParser.mjs 5:11
Module parse failed: Unexpected token (5:11)
File was processed with these loaders:
 * ./node_modules/babel-loader/lib/index.js
You may need an additional loader to handle the result of these loaders.
| import { parseAnyDigitsSigned } from "../utils.mjs";
| export class TimestampMillisecondsParser extends Parser {
>   priority = 20;
|   parse(dateString) {
|     return parseAnyDigitsSigned(dateString);
 @ ./node_modules/date-fns/parse/_lib/parsers.mjs 31:0-88 108:9-36
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers/TimestampSecondsParser.mjs 5:11
Module parse failed: Unexpected token (5:11)
File was processed with these loaders:
 * ./node_modules/babel-loader/lib/index.js
You may need an additional loader to handle the result of these loaders.
| import { parseAnyDigitsSigned } from "../utils.mjs";
| export class TimestampSecondsParser extends Parser {
>   priority = 40;
|   parse(dateString) {
|     return parseAnyDigitsSigned(dateString);
 @ ./node_modules/date-fns/parse/_lib/parsers.mjs 30:0-78 107:9-31
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers/Hour0To11Parser.mjs 5:11
Module parse failed: Unexpected token (5:11)
File was processed with these loaders:
 * ./node_modules/babel-loader/lib/index.js
You may need an additional loader to handle the result of these loaders.
| import { parseNDigits, parseNumericPattern } from "../utils.mjs";
| export class Hour0To11Parser extends Parser {
>   priority = 70;
|   parse(dateString, token, match) {
|     switch (token) {
 @ ./node_modules/date-fns/parse/_lib/parsers.mjs 23:0-64 100:9-24
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers/Hour0to23Parser.mjs 5:11
Module parse failed: Unexpected token (5:11)
File was processed with these loaders:
 * ./node_modules/babel-loader/lib/index.js
You may need an additional loader to handle the result of these loaders.
| import { parseNDigits, parseNumericPattern } from "../utils.mjs";
| export class Hour0to23Parser extends Parser {
>   priority = 70;
|   parse(dateString, token, match) {
|     switch (token) {
 @ ./node_modules/date-fns/parse/_lib/parsers.mjs 22:0-64 99:9-24
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers/Hour1To24Parser.mjs 5:11
Module parse failed: Unexpected token (5:11)
File was processed with these loaders:
 * ./node_modules/babel-loader/lib/index.js
You may need an additional loader to handle the result of these loaders.
| import { parseNDigits, parseNumericPattern } from "../utils.mjs";
| export class Hour1To24Parser extends Parser {
>   priority = 70;
|   parse(dateString, token, match) {
|     switch (token) {
 @ ./node_modules/date-fns/parse/_lib/parsers.mjs 24:0-64 101:9-24
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers/Hour1to12Parser.mjs 5:11
Module parse failed: Unexpected token (5:11)
File was processed with these loaders:
 * ./node_modules/babel-loader/lib/index.js
You may need an additional loader to handle the result of these loaders.
| import { parseNDigits, parseNumericPattern } from "../utils.mjs";
| export class Hour1to12Parser extends Parser {
>   priority = 70;
|   parse(dateString, token, match) {
|     switch (token) {
 @ ./node_modules/date-fns/parse/_lib/parsers.mjs 21:0-64 98:9-24
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers/MinuteParser.mjs 5:11
Module parse failed: Unexpected token (5:11)
File was processed with these loaders:
 * ./node_modules/babel-loader/lib/index.js
You may need an additional loader to handle the result of these loaders.
| import { parseNDigits, parseNumericPattern } from "../utils.mjs";
| export class MinuteParser extends Parser {
>   priority = 60;
|   parse(dateString, token, match) {
|     switch (token) {
 @ ./node_modules/date-fns/parse/_lib/parsers.mjs 25:0-58 102:9-21
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers/SecondParser.mjs 5:11
Module parse failed: Unexpected token (5:11)
File was processed with these loaders:
 * ./node_modules/babel-loader/lib/index.js
You may need an additional loader to handle the result of these loaders.
| import { parseNDigits, parseNumericPattern } from "../utils.mjs";
| export class SecondParser extends Parser {
>   priority = 50;
|   parse(dateString, token, match) {
|     switch (token) {
 @ ./node_modules/date-fns/parse/_lib/parsers.mjs 26:0-58 103:9-21
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/Setter.mjs 5:14
Module parse failed: Unexpected token (5:14)
File was processed with these loaders:
 * ./node_modules/babel-loader/lib/index.js
You may need an additional loader to handle the result of these loaders.
| const TIMEZONE_UNIT_PRIORITY = 10;
| export class Setter {
>   subPriority = 0;
|   validate(_utcDate, _options) {
|     return true;
 @ ./node_modules/date-fns/parse.mjs 8:0-69 352:23-49
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers/MonthParser.mjs 5:21
Module parse failed: Unexpected token (5:21)
File was processed with these loaders:
 * ./node_modules/babel-loader/lib/index.js
You may need an additional loader to handle the result of these loaders.
| import { mapValue, parseNDigits, parseNumericPattern } from "../utils.mjs";
| export class MonthParser extends Parser {
>   incompatibleTokens = ["Y", "R", "q", "Q", "L", "w", "I", "D", "i", "e", "c", "t", "T"];
|   priority = 110;
|   parse(dateString, token, match) {
 @ ./node_modules/date-fns/parse/_lib/parsers.mjs 8:0-56 85:9-20
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers/DayParser.mjs 6:11
Module parse failed: Unexpected token (6:11)
File was processed with these loaders:
 * ./node_modules/babel-loader/lib/index.js
You may need an additional loader to handle the result of these loaders.
| // Day of week
| export class DayParser extends Parser {
>   priority = 90;
|   parse(dateString, token, match) {
|     switch (token) {
 @ ./node_modules/date-fns/parse/_lib/parsers.mjs 14:0-52 91:9-18
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers/DayPeriodParser.mjs 6:11
Module parse failed: Unexpected token (6:11)
File was processed with these loaders:
 * ./node_modules/babel-loader/lib/index.js
You may need an additional loader to handle the result of these loaders.
| // in the morning, in the afternoon, in the evening, at night
| export class DayPeriodParser extends Parser {
>   priority = 80;
|   parse(dateString, token, match) {
|     switch (token) {
 @ ./node_modules/date-fns/parse/_lib/parsers.mjs 20:0-64 97:9-24
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers/ISODayParser.mjs 7:11
Module parse failed: Unexpected token (7:11)
File was processed with these loaders:
 * ./node_modules/babel-loader/lib/index.js
You may need an additional loader to handle the result of these loaders.
| // ISO day of week
| export class ISODayParser extends Parser {
>   priority = 90;
|   parse(dateString, token, match) {
|     const valueCallback = value => {
 @ ./node_modules/date-fns/parse/_lib/parsers.mjs 17:0-58 94:9-21
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers/LocalDayParser.mjs 7:11
Module parse failed: Unexpected token (7:11)
File was processed with these loaders:
 * ./node_modules/babel-loader/lib/index.js
You may need an additional loader to handle the result of these loaders.
| // Local day of week
| export class LocalDayParser extends Parser {
>   priority = 90;
|   parse(dateString, token, match, options) {
|     const valueCallback = value => {
 @ ./node_modules/date-fns/parse/_lib/parsers.mjs 15:0-62 92:9-23
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers/StandAloneLocalDayParser.mjs 7:11
Module parse failed: Unexpected token (7:11)
File was processed with these loaders:
 * ./node_modules/babel-loader/lib/index.js
You may need an additional loader to handle the result of these loaders.
| // Stand-alone local day of week
| export class StandAloneLocalDayParser extends Parser {
>   priority = 90;
|   parse(dateString, token, match, options) {
|     const valueCallback = value => {
 @ ./node_modules/date-fns/parse/_lib/parsers.mjs 16:0-82 93:9-33
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers/ISOWeekYearParser.mjs 8:11
Module parse failed: Unexpected token (8:11)
File was processed with these loaders:
 * ./node_modules/babel-loader/lib/index.js
You may need an additional loader to handle the result of these loaders.
| // ISO week-numbering year
| export class ISOWeekYearParser extends Parser {
>   priority = 130;
|   parse(dateString, token) {
|     if (token === "R") {
 @ ./node_modules/date-fns/parse/_lib/parsers.mjs 4:0-68 81:9-26
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers/LocalWeekYearParser.mjs 8:11
Module parse failed: Unexpected token (8:11)
File was processed with these loaders:
 * ./node_modules/babel-loader/lib/index.js
You may need an additional loader to handle the result of these loaders.
| // Local week-numbering year
| export class LocalWeekYearParser extends Parser {
>   priority = 130;
|   parse(dateString, token, match) {
|     const valueCallback = year => ({
 @ ./node_modules/date-fns/parse/_lib/parsers.mjs 3:0-72 80:9-28
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers/DateParser.mjs 9:11
Module parse failed: Unexpected token (9:11)
File was processed with these loaders:
 * ./node_modules/babel-loader/lib/index.js
You may need an additional loader to handle the result of these loaders.
| // Day of the month
| export class DateParser extends Parser {
>   priority = 90;
|   subPriority = 1;
|   parse(dateString, token, match) {
 @ ./node_modules/date-fns/parse/_lib/parsers.mjs 12:0-54 89:9-19
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers/ISOWeekParser.mjs 9:11
Module parse failed: Unexpected token (9:11)
File was processed with these loaders:
 * ./node_modules/babel-loader/lib/index.js
You may need an additional loader to handle the result of these loaders.
| // ISO week of year
| export class ISOWeekParser extends Parser {
>   priority = 100;
|   parse(dateString, token, match) {
|     switch (token) {
 @ ./node_modules/date-fns/parse/_lib/parsers.mjs 11:0-60 88:9-22
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers/LocalWeekParser.mjs 9:11
Module parse failed: Unexpected token (9:11)
File was processed with these loaders:
 * ./node_modules/babel-loader/lib/index.js
You may need an additional loader to handle the result of these loaders.
| // Local week of year
| export class LocalWeekParser extends Parser {
>   priority = 100;
|   parse(dateString, token, match) {
|     switch (token) {
 @ ./node_modules/date-fns/parse/_lib/parsers.mjs 10:0-64 87:9-24
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers/ISOTimezoneParser.mjs 9:11
Module parse failed: Unexpected token (9:11)
File was processed with these loaders:
 * ./node_modules/babel-loader/lib/index.js
You may need an additional loader to handle the result of these loaders.
| // Timezone (ISO-8601)
| export class ISOTimezoneParser extends Parser {
>   priority = 10;
|   parse(dateString, token) {
|     switch (token) {
 @ ./node_modules/date-fns/parse/_lib/parsers.mjs 29:0-68 106:9-26
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js

ERROR in ./node_modules/date-fns/parse/_lib/parsers/ISOTimezoneWithZParser.mjs 9:11
Module parse failed: Unexpected token (9:11)
File was processed with these loaders:
 * ./node_modules/babel-loader/lib/index.js
You may need an additional loader to handle the result of these loaders.
| // Timezone (ISO-8601. +00:00 is `'Z'`)
| export class ISOTimezoneWithZParser extends Parser {
>   priority = 10;
|   parse(dateString, token) {
|     switch (token) {
 @ ./node_modules/date-fns/parse/_lib/parsers.mjs 28:0-78 105:9-31
 @ ./node_modules/date-fns/parse.mjs
 @ ./node_modules/date-fns/index.mjs
 @ ./src/utcToZonedTime/test.js
 @ ./src sync \/test\.js$
 @ ./test.js
ℹ ｢wdm｣: Failed to compile.
08 01 2024 08:58:11.335:INFO [karma-server]: Karma v6.4.2 server started at http://localhost:9876/
08 01 2024 08:58:11.335:INFO [launcher]: Launching browsers LocalChrome with concurrency unlimited
08 01 2024 08:58:11.339:INFO [launcher]: Starting browser Chrome
08 01 2024 08:58:15.698:INFO [Chrome 120.0.0.0 (Mac OS 10.15.7)]: Connected on socket M-sl7g7Es8dIyBjSAAAB with id 52467360
Chrome 120.0.0.0 (Mac OS 10.15.7) ERROR
  Uncaught Error: Module parse failed: Unexpected token (3:11)
  File was processed with these loaders:
   * ./node_modules/babel-loader/lib/index.js
  You may need an additional loader to handle the result of these loaders.
  | import { Parser } from "../Parser.mjs";
  | export class EraParser extends Parser {
  >   priority = 140;
  |   parse(dateString, token, match) {
  |     switch (token) {
  at /Users/christopherklint/coding/date-fns-tz/test.js:21074:1

  Error: Module parse failed: Unexpected token (3:11)
  File was processed with these loaders:
   * ./node_modules/babel-loader/lib/index.js
  You may need an additional loader to handle the result of these loaders.
  | import { Parser } from "../Parser.mjs";
  | export class EraParser extends Parser {
  >   priority = 140;
  |   parse(dateString, token, match) {
  |     switch (token) {
      at ./node_modules/date-fns/parse/_lib/parsers/EraParser.mjs (/Users/christopherklint/coding/date-fns-tz/test.js:21074:7)
      at __webpack_require__ (webpack:///webpack/bootstrap:19:1 <- /Users/christopherklint/coding/date-fns-tz/test.js:20:30)
      at ./node_modules/date-fns/parse/_lib/parsers.mjs (webpack:///node_modules/date-fns/parse/_lib/parsers.mjs:1:1 <- /Users/christopherklint/coding/date-fns-tz/test.js:20858:80)
      at __webpack_require__ (webpack:///webpack/bootstrap:19:1 <- /Users/christopherklint/coding/date-fns-tz/test.js:20:30)
      at ./node_modules/date-fns/parse.mjs (webpack:///node_modules/date-fns/parse.mjs:1:1 <- /Users/christopherklint/coding/date-fns-tz/test.js:20387:80)
      at __webpack_require__ (webpack:///webpack/bootstrap:19:1 <- /Users/christopherklint/coding/date-fns-tz/test.js:20:30)
      at ./node_modules/date-fns/isMatch.mjs (webpack:///node_modules/date-fns/isMatch.mjs:1:1 <- /Users/christopherklint/coding/date-fns-tz/test.js:15953:68)
      at __webpack_require__ (webpack:///webpack/bootstrap:19:1 <- /Users/christopherklint/coding/date-fns-tz/test.js:20:30)
      at ./node_modules/date-fns/index.mjs (webpack:///node_modules/date-fns/index.mjs:1:1 <- /Users/christopherklint/coding/date-fns-tz/test.js:14389:72)
      at __webpack_require__ (webpack:///webpack/bootstrap:19:1 <- /Users/christopherklint/coding/date-fns-tz/test.js:20:30)

Finished in 0.138 secs / 0 secs @ 08:58:15 GMT+0100 (GMT+01:00)

SUMMARY:
✔ 0 tests completed
```